worldcup/config.py

```diff
@@ -1,14 +1,22 @@
 # coding: utf-8
 
 import os
+from collections import namedtuple
 
+Tournament = namedtuple('Tournament', ['dbname', 'league'])
 
 SECRET_KEY = os.getenv('SECRET_KEY', 'super secret key')
 
 MONGO_URI = os.getenv('MONGO_URI', 'mongodb://localhost:27017/')
-MONGO_DBNAME = os.getenv('MONGO_DBNAME', 'worldcup')
+MONGO_LOGINDB = os.getenv('MONGO_LOGINDB', 'logins')
 
 WECHAT_APPID = os.getenv('WECHAT_APPID', '')
 WECHAT_APPSECRET = os.getenv('WECHAT_APPSECRET', '')
 
-LEAGUE_NAME = '世界杯'
+DEFAULT_TOURNAMENT = Tournament('worldcup2018', '世界杯')
```

Comment:
少点重复代码，换 default 的时候也好换

```suggestion
DEFAULT_TOURNAMENT = TOURNAMENTS[-2]
```